core/pom.xml

```diff
@@ -43,6 +43,11 @@
             <groupId>io.confluent</groupId>
             <artifactId>rest-utils</artifactId>
         </dependency>
+        <dependency>
+            <groupId>io.confluent</groupId>
+            <artifactId>kafka-client-plugins</artifactId>
+            <version>${ce.kafka.version}</version>
```

Comment:
@lucy-fan , the dependency with the version should go in the dependencyManagement section of the root pom.xml (https://github.com/confluentinc/schema-registry/blob/master/pom.xml) and then the version here should be omitted

Reply:
I've added the dependency with the version to the root pom.xml and removed the version from the dependency here!